### ISSUE TITLE
Contact person and about-page spaces

### ISSUE THREAD

**Issue by erlend-sh** [commune-os/commune-server#25]
Spaces need a basic _about_ page that lists at minimum a contact person who’s responsible for the instance, but ideally some additional information like activity metrics et.al.

#### Examples

https://mastodon.social/about
![IMG_2107](https://github.com/commune-os/commune-server/assets/583842/70475c09-767b-4f2c-a9fb-7ff263845b96)

https://socialhub.activitypub.rocks/about
![IMG_2108](https://github.com/commune-os/commune-server/assets/583842/3c9cfe89-792b-4b68-9fe5-b44aa3cc347a)

#### Related Matrix specs
- https://github.com/matrix-org/matrix-spec-proposals/pull/1929
- https://github.com/matrix-org/matrix-spec-proposals/pull/2063

**Comment by ChurchOfTheSubgenius** [comment]
> Spaces need a basic about page that lists at minimum a contact person who’s responsible for the instance, but ideally some additional information like activity metrics et.al.

Do you mean each individual space, or the entire app/instance? I suppose it makes sense for both to have an about page.

**Comment by erlend-sh** [comment]
Yeah, both are needed in the case of commune. The linked matrix specs are specifically about homeservers though, not spaces.
